app/views/spree/products/_description.html.erb

```diff
@@ -1,22 +1,37 @@
-<h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%= Spree.t(:description) %></h3>
+<!-- <h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%#= Spree.t(:description) %></h3>
 
-<% if Spree::Config[:show_raw_product_description] || product_wysiwyg_editor_enabled? %>
+<%# if Spree::Config[:show_raw_product_description] || product_wysiwyg_editor_enabled? %>
   <div id="product-description-long" class="m-0 text-break product-description" data-hook="description">
-    <%= raw product_description(@product) %>
+    <%#= raw product_description(@product) %>
   </div>
-<% else %>
+<%# else %>
   <div id="product-description-short" class="m-0 text-break product-description" data-hook="short-description">
-      <%= sanitize product_description(@product).truncate(450) %>
+      <%#= sanitize product_description(@product).truncate(450) %>
   </div>
   <div id="product-description-long" class="m-0 text-break product-description d-none" data-hook="description">
-    <%= sanitize product_description(@product) %>
+    <%#= sanitize product_description(@product) %>
   </div>
-  <% if product_description(@product).length > 450 %>
+  <%# if product_description(@product).length > 450 %>
     <span id="product-description-arrow" class="d-flex justify-content-center align-items-center mt-3 mx-auto product-description-arrow" aria-hidden="true">
-      <%= icon(name: 'arrow-right',
+      <%#= icon(name: 'arrow-right',
                classes: 'spree-icon-arrow spree-icon-arrow-down',
                width: 20,
                height: 20) %>
-    </span>
+    </span> -->
+  <%# end %>
+<%# end %>
+<% if Spree::Config[:show_raw_product_description] || product_wysiwyg_editor_enabled? %>
```

Comment:
As per default spree, this check should add on long description but we added it over short description. Please remove this check from here and add over long description. See default spree reference
<img width="1002" alt="Screen Shot 2021-12-09 at 1 37 39 AM" src="https://user-images.githubusercontent.com/77246192/145280540-052765ae-5f63-4ddb-9094-0a4d0e94092f.png">